sdk/core/Azure.Core/tests/TestClients/TestResourceOperationOrResponse.cs

```diff
@@ -12,7 +12,7 @@ internal class TestResourceOperationOrResponse : Operation
     {
         private TestResource _value;
         private bool _exceptionOnWait;
-        private OperationOrResponseInternals<TestResource> _operationHelper;
+        private OperationInternal<TestResource> _operationHelper;
         private int _delaySteps = 0;
 
         protected TestResourceOperationOrResponse()
```

Comment:
Should the name of the test class/test methods be changed so that they reflect which type is being tested?

Reply:
We don't test `OperationOrResponseInternals` with this type. This type is a mock.
@m-nash, what do you think about naming?

Reply:
I would imagine if we are consolidating on Operation and getting rid of OperationOrResponse we should get rid of these classes as well and consolidate the tests.

Reply:
We can't get rid of these test classes cause ARM still generates individual `Operation` types (one per project). But I've changed the name.